recipes/vc-feature/meta.yaml

```diff
@@ -0,0 +1,23 @@
+{% set version = "0.0.1" %}
```

Comment:
We would want to select the version of `vc` that we build against. Setting it as the package version allows us to do this easily in other recipes.